src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -72,36 +72,19 @@ class ResourceTemplateForm extends Component {
     }
   }
 
-  getContentForModalButton = (rtId) => {
```

Comment:
Dead code, AFAICT. Not invoked anywhere in the codebase.

Reply:
This is removed in PR #453